.github/workflows/CI.yml

```diff
@@ -41,26 +41,34 @@ jobs:
           ls ./src/test/resources/org/opensearch/ad/bwc/anomaly-detection/$plugin_version
       - name: Build and Run Tests
         run: |
-          ./gradlew build -Dtest.logs=true
+          ./gradlew assemble
```

Comment:
assemble will not run unit and integ tests. Also, I added test.logs=true to print out precision/recall of AD. Why do you need to replace build with assemble?